frontend/index.html

```diff
@@ -1,23 +1,20 @@
 <!doctype html>
-<html lang="eng">
+<html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="x-ua-compatible" content="ie=edge">
     <title>Group Income</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
     <link rel="stylesheet" href="/assets/css/main.css">
     <link rel="icon" href="/assets/images/group-income-icon-transparent.png" sizes="32x32">
   </head>
-  <body lang="eng">
-    <!--[if lt IE 9]>
-    <div class="modal is-active">
-        <div class="modal-content">
-            <h1>Outdated browser</h1>
-            <p>You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        </div>
+  <body>
+    <div class="modal is-active" id="fallback" style="visibility: hidden;">
+      <div class="modal-content">
+        <h1>Outdated browser</h1>
+        <p>You are using an <strong>outdated</strong> or unsupported browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
```

Comment:
Nice update! I have one request: at first I thought that we should change this to be an https link instead of http, but on second thought I think it would be better to simply remove the link. We don't know how long this website will stay up for, we don't know whether or not they'll get compromised and start serving malware, and we don't know if they'll start recommending bad browsers, so IMO it would be better to avoid all these possibilities and leave it at, "please upgrade your browser".

Reply:
Agreed :+1: